packages/dds/merge-tree/src/collections/index.ts

```diff
@@ -0,0 +1,11 @@
+/*!
```

Comment:
merge-tree/src/index.ts currently exports * from ./collections.  Should that be scoped some, if the intent is to limit visibility outside the package?

Reply:
it will be. although i don't think it should break anyone i'm not unilaterally removing packages in main. i'll clean up the exports in next, so if anyone happens to be using them they get a deprecation window.

Reply:
the move to individual files makes it easier to turn on/off per file, while not hitting merge conflicts between main and next